src/handlers/proposalsHandlers.ts

```diff
@@ -43,26 +43,19 @@ const getProposals = (
   next: NextFunction,
 ): void => {
   const paginationParameters = extractPaginationParameters(req);
-  db.sql(
-    'proposals.selectWithPagination',
-    {
-      ...getLimitValues(paginationParameters),
-    },
-  )
-    .then((proposalsQueryResult: Result<Proposal>) => {
-      logger.debug(proposalsQueryResult);
-      const { rows: proposals } = proposalsQueryResult;
-      if (isProposalArray(proposals)) {
-        res.status(200)
-          .contentType('application/json')
-          .send(proposals);
-      } else {
-        next(new InternalValidationError(
-          'The database responded with an unexpected format.',
-          isProposalArray.errors ?? [],
-        ));
-      }
-    })
+  (async () => {
+    const proposalBundle = await loadBundle(
+      'proposals.selectWithPagination',
+      {
+        ...getLimitValues(paginationParameters),
+      },
+      'proposals',
+      isProposal,
+    );
```

Comment:
I wonder if this is worth promoting into a named, exported function... we're only doing it twice - and only once in production code - but that's getting to be a lot of stuff that needs to be kept in sync, and it kind of feels like not the handler's responsibility for that.